tasks/linter.py

```diff
@@ -156,7 +157,16 @@ def go(
         inv linter.go --targets=./pkg/collector/check,./pkg/aggregator
         inv linter.go --module=.
     """
-    if not check_tools_version(ctx, ['go', 'golangci-lint']):
+    if not check_tools_version(ctx, ['golangci-lint']):
+        print(
+            color_message(
+                "Error: The golanci-lint version you are using is not the corect one. Please run inv -e install-tools to install the correct version.",
+                "red",
+            )
+        )
+        raise Exit(code=1)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏The message can be wrapped within `Exit`